Dockerfile

```diff
@@ -42,6 +42,8 @@ RUN npm run build
 
 
 FROM base AS production
+RUN --mount=type=cache,target=/root/.cache/uv \
+  uv sync --locked --group prod
```

Comment:
I suspected that this might be needed. Good catch!
I was wondering about that group, is there any benefit to having it separated? Could we just move the dependencies into the normal list?

Reply:
If so, this step could be skipped